inc/field/fieldsfield.class.php

```diff
@@ -152,16 +144,23 @@ public function getBlocks(): array {
    }
 
    public function showForm(array $options): void {
+      if (!\Plugin::isPluginActive('fields')) {
+         $template = '@formcreator/field/undefinedfield.html.twig';
+         TemplateRenderer::getInstance()->display($template, [
+            'item' => $this->question,
+            'params' => $options,
+         ]);
+         return;
+      }
```

Comment:
Ensure we can show the modal for the question and edit it even when Fields is disabled